composer.lock

```diff
@@ -7007,5 +7011,5 @@
         "ext-json": "*"
     },
     "platform-dev": [],
-    "plugin-api-version": "1.1.0"
+    "plugin-api-version": "2.0.0"
```

Comment:
composer 2?

Reply:
yes, the latest tag now points to v2: https://hub.docker.com/_/composer